src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt

```diff
@@ -125,10 +127,17 @@ class NexusPublishPlugin : Plugin<Project> {
                 enabled = false
             }
         }
-        rootProject.tasks.register<Task>(SIMPLIFIED_CLOSE_AND_RELEASE_TASK_NAME) {
+        rootProject.tasks.register(SIMPLIFIED_CLOSE_AND_RELEASE_TASK_NAME) {
             group = PublishingPlugin.PUBLISH_TASK_GROUP
             description = "Closes and releases open staging repositories in all configured Nexus instances."
-            enabled = false
```

Comment:
What was this `enabled = false/true` about? Is it safe to remove, or we need it for the other ones too?

Reply:
It is a task without actions, it never does anything anyway. Disabling or enabling it is pointless.

Reply:
Could enabling it yield to some more tasks executed (via dependsOn) and therefore breaking some builds which just do blanket `gradlew closeAndReleaseStagingRepository` in a multimodule build?

Reply:
Was it maybe disabled so that `gradlew tasks` doesn't list it when there are no repositories registered?

Reply:
A no to both, but you are welcome to come up with other options I might have forgotten. :-)

Setting a task to `enabled = false` does not prevent its dependencies to run.
They are still run, just the task action itself is skipped and there is none here.
To skip a task and its dependencies (if nothing else depends on them) you have to use `-x theTask` or change `gradle.startParameter.excludedTaskNames` which is the same.
The `enabled` property on a task without actions is really just a no-op.

There is only one exception actually now that I think of it.
If someone got the idea to register additional actions on the task using `doLast` or `doFirst` then it would make a difference.
But in reality the tasks are never disabled anyway, because why should someone apply the plugin and then not define any repositories. :-)

> Was it maybe disabled so that gradlew tasks doesn't list it when there are no repositories registered?

Not in any Gradle version I am aware of, and that is 1.0-milestone-4 and newer iirc. :-)
```plain
Publishing tasks
----------------
closeAndReleaseStagingRepository - Closes and releases open staging repositories in all configured Nexus instances.
```
You see from the description that it was not adjusted for listing the repositories and this is with the latest published version and only applying it without configuration.